protos/capture.proto

```diff
@@ -218,3 +218,25 @@ message CaptureHeader {
   uint32 num_timers = 2;
   uint32 version = 3;
 }
+
+message TimerData {
```

Comment:
In my opinion, timers should be completely re-thought instead of simply translate them to protobufs. So far we've always used "timers" as a catch all for anything that has a start and an end, with some fields that apply to some type but not others in a very confusing way. This is not a nice design. If we have different types of time slices (e.g., scheduling slices, dynamic instrumentation, gpu driver events, etc.) we should just have different types of protobufs, similarly to what is done for the gRPC capture messages.